.gitpod.yml

```
@@ -10,6 +10,12 @@ ports:
 - port: 3000
   onOpen: ignore
 tasks:
+- name: watch-js
+  # This task watches for changes to JS files and automatically builds
+  init: gp sync-await docker-up && docker-compose run --rm home npm run-script watch
+- name: watch-css
+  # This task watches for changes to CSS files and automatically builds
+  init: gp sync-await docker-up && docker-compose run --rm home npm run-script watch-css
```

Can you rebase and test if this works? Concurrently is now in the package.json :)

```suggestion
- name: watch
  # This task watches for changes to JS/CSS files and automatically builds
  init: gp sync-await docker-up && docker-compose run --rm home npx concurrently npm:watch npm:watch-css
```

If it errors, might need to wait for the next deploy until the olbase image is regenerated